charts/dendrite/values.yaml

```diff
@@ -0,0 +1,599 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default
+# values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+---
+image:
+  # -- image repository
+  repository: ghcr.io/matrix-org/dendrite-monolith
+  # -- image tag
+  # @default -- chart.appVersion
+  tag:
+  # -- image pull policy
+  pullPolicy: IfNotPresent
+
+# -- If added dendrite will start a HTTP and HTTPS listener
+# args:
+#   - "--tls-cert=server.crt"
+#   - "--tls-key=server.key"
+
+# -- Configures service settings for the chart.
+# @default -- See values.yaml
+service:
+  main:
+    ports:
+      # -- Configures the default HTTP listener for dendrite
+      # @default -- See values.yaml
+      http:
+        port: 8008
+      # -- Configures the HTTPS listener for dendrite
+      # @default -- See values.yaml
+      https:
+        enabled: true
+        port: 8448
+        protocol: HTTPS
+
+ingress:
+  # -- (Monolith Only) Enable and configure ingress settings for the chart under
+  # this key.
+  # @default -- See values.yaml
+  main:
+    enabled: false
```

Comment:
- missing `annotations` for `certmanager` and other parts tooling in a k8s
- missing `tls` for store `certs`
- missing `ingressClassName` for multiple ingress setups in k8s

Reply:
I believe you have some of them for a poly setup (but not documented in values.yaml)
And for a monolith setup, i do not see an ingress

Reply:
This is quite complicated chart, so please see the dependencies as quite a lot of those missing things come from there, unless you mean that a user wouldn't need to specify them separately?

The ingress is currently ONLY for a monolith deployment due to the complexities of the polylith one.

I'm currently utilizing this here (using my own chart repo and modified chart): https://github.com/samip5/k8s-cluster/blob/main/cluster/apps/default/dendrite/helm-release.yaml